wsi/drm/drmdisplay.h

```diff
@@ -215,6 +218,7 @@ class DrmDisplay : public PhysicalDisplay {
   std::vector<drmModeModeInfo> modes_;
   SpinLock display_lock_;
   DrmDisplayManager *manager_;
+  uint32_t vsync_period_ = 0;
```

Comment:
I suppose we don't need a new property “vsync_period_ ” here since it is available from modes_[config_]
current_config is already saved.